templates/slide.html

```diff
@@ -420,23 +435,44 @@ <h2 class="popover__title">{{disp_str[i]}}</h2>
  function onPrevPage() {
    // empty previous child elements of textLayer
    $("#textLayerid").empty();
-
+   course = "{{course_name}}";
+    lec = "{{lec_name}}";
    if (pageNum <= 1) {
      return;
    }
+  //Added updated_slide to keep track of slide name
    pageNum--;
+   new_slide_name= "{{course_name}}".concat("---").concat("{{lec_name}}").concat("---slide").concat((pageNum-1).toString());
```

Comment:
Will it be possible to call one function to log since we are using it many times throughout this file 

for example - 

```
log_action(updated_slide,action,route){

logdata = JSON.stringify({
        action: 'prev',
        updated_slide: new_slide_name, 
        route: window.location.pathname
    });
    navigator.sendBeacon('{{base_url}}log_action', logdata);
    
    }
```

Then we can call this log_action function from where we need- with the appropriate parameters.